src/pke/lib/schemerns/rns-pke.cpp

```diff
@@ -96,12 +96,11 @@ DecryptResult PKERNS::Decrypt(ConstCiphertext<DCRTPoly> ciphertext, const Privat
 
     b.SetFormat(Format::COEFFICIENT);
     const size_t sizeQl = b.GetParams()->GetParams().size();
-
-    if (sizeQl == 1)
-        *plaintext = b.GetElementAtIndex(0);
-    else
+    if (sizeQl != 1)
```

Comment:
Seems like the same issue as above. Make the same change.

Reply:
made the same change described above